functional/SceneGraphTests/src/test/scenegraph/richtext/RichTextPropertiesApp.java

```diff
@@ -242,9 +244,16 @@ public void selectFlow(int num) {
     public void addFlow() {
         textFlowPage.addTextFlow.fire();
     }
+
     public void requestDefaultFocus(){
-        textFlowPage.addTextFlow.requestFocus();
-    }
+
+        PlatformImpl.runAndWait(new Runnable() {
+            @Override
+            public void run() {
+                textFlowPage.addTextFlow.requestFocus();
+            }
+        });
+   }
```

Comment:
minor : can be replaced with a lambda: 
 PlatformImpl.runAndWait(() ->  textFlowPage.addTextFlow.requestFocus());